lib/index.js

```diff
@@ -247,23 +247,28 @@ api.createSignatureString = ({includeHeaders, requestOptions} = {}) => {
  * the request creating a signature & AuthzHeader.
  *
  * @param {Object} request - A request object.
- * @param {Object} options - Options for parsing such as clockSkew.
+ * @param {Object} [options={}] - Options for parsing such as clockSkew.
  *
  * @returns {Object} A parsed request with a signed header.
 */
 api.parseRequest = (request, options) => {
```

Comment:
```suggestion
api.parseRequest = (request, options = {headers: ['(created)']}) => {
```

Reply:
I think `options = {}` would be better here as `options.clockSkew` and `options.headers` are added later.

Reply:
this is similar, but the default does not contain `headers`: https://github.com/digitalbazaar/http-signature-header/pull/24/commits/8dc1062bfb161d3ca74fdf8318140d8a0676dbfa